ext/native/thin3d/thin3d_gl.cpp

```diff
@@ -1530,7 +1531,7 @@ void OpenGLContext::fbo_unbind() {
 	currentReadHandle_ = 0;
 }
 
-void OpenGLContext::BindFramebufferAsRenderTarget(Framebuffer *fbo) {
+void OpenGLContext::BindFramebufferAsRenderTarget(Framebuffer *fbo, const RenderPassInfo &rp) {
 	CHECK_GL_ERROR_IF_DEBUG();
 	if (fbo) {
 		OpenGLFramebuffer *fb = (OpenGLFramebuffer *)fbo;
```

Comment:
Most drivers will interpretr CLEAR as an implicit invalidation (well, they'll just clear on loading into the tilebuffer so it doesn't matter) but yes, for don't-care it probably makes sense. I'll leave that for later though.